infra/recipes/docker-compose/common/aws-services.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.3"
+services:
+  minioS3:
+    image: minio/minio
+    environment:
+      - MINIO_ROOT_USER=admin
+      - MINIO_ROOT_PASSWORD=password
+    ports:
+      - 9871:9001
+      - 9870:9000
+    command: [ "server", "/data", "--console-address", ":9001" ]
+  mc:
+    depends_on:
+      - minioS3
+    image: minio/mc
+    environment:
+      - AWS_ACCESS_KEY_ID=admin
+      - AWS_SECRET_ACCESS_KEY=password
+      - AWS_REGION=us-east-1
+    entrypoint: >
+      /bin/sh -c "
+      until (/usr/bin/mc config host add minio http://minioS3:9000 admin password) do echo '...waiting...' && sleep 1; done;
```

Comment:
So looks like `9000` is the actual service port.

Reply:
I want to refrain adding e2e documentation in this PR. I still want to think about the write place to do it. added a yml comment for now.